engine/common/host.c

```diff
@@ -1070,3 +1082,26 @@ BOOL WINAPI DllMain( HINSTANCE hinstDLL, DWORD fdwReason, LPVOID lpvReserved )
 	return TRUE;
 #endif
 }
+
+#ifdef PANDORA
+void Pandora_Host_Shutdown( void )
+{
+	if( host.shutdown_issued ) return;
+	host.shutdown_issued = true;
+
+	if( host.state != HOST_ERR_FATAL ) host.state = HOST_SHUTDOWN; // prepare host to normal shutdown
+	if( !host.change_game ) Q_strncpy( host.finalmsg, "Server shutdown", sizeof( host.finalmsg ));
+
+	if( host.type == HOST_NORMAL )
+		Host_WriteConfig();
+
+	SV_Shutdown( false );
+	CL_Shutdown();
+
+	Mod_Shutdown();
+	NET_Shutdown();
+	Host_FreeCommon();
+	Con_DestroyConsole();
+
+}
+#endif

```

Comment:
I have to test if it's the same bug and if it's fixed with the other commit.
Even just quitting the game trigger a bug on my side, and I needed another Name for the function.